udata/features/transfer/api.py

```diff
@@ -81,14 +85,34 @@
 
 ns = api.namespace("transfer")
 
+requests_parser = api.parser()
+requests_parser.add_argument(
+    "subject", type=str, help="ID of dataset, dataservice, reuse…", location="args"
+)
+requests_parser.add_argument(
+    "recipient", type=str, help="ID of user or organization", location="args"
+)
+
 
 @ns.route("/", endpoint="transfers")
 class TransferRequestsAPI(API):
     @api.doc("list_transfers")
     @api.marshal_list_with(transfer_fields)
     def get(self):
-        """List all transfer requests"""
-        pass
+        args = requests_parser.parse_args()
+
+        transfers = Transfer.objects
```

Comment:
Don't we want to filter on `Transfer.objects(status="pending")` by default for example?
Else we have more than 60K existing transfer objects

Reply:
For now, I added a check for at least subject or recipient to prevent big responses d02b8a8633f1b6d249ebe876b7d2294942379444